CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - [#420](https://github.com/embedded-graphics/embedded-graphics/pull/420) Added support for `SubImage`s.
 - [#429](https://github.com/embedded-graphics/embedded-graphics/pull/429) Added `ToBytes` trait to convert colors into byte arrays.
 - [#431](https://github.com/embedded-graphics/embedded-graphics/pull/431) Added `MockDisplay::affected_area` to get the area affected by previous drawing operations.
+- [#439](https://github.com/embedded-graphics/embedded-graphics/pull/439) Added support to render transparent characters.
```

Comment:
IMO it's important to mention the background, because otherwise it might not be clear why someone wants to render transparent text:
```suggestion
- [#439](https://github.com/embedded-graphics/embedded-graphics/pull/439) Added support to render transparent characters on a colored background.
```